alpaca/data/enums.py

```diff
@@ -94,6 +94,20 @@ class CryptoFeed(str, Enum):
     US = "us"
 
 
+class OptionsFeed(str, Enum):
+    """
+    The source feed of the data. `opra` is the official OPRA feed,
+    `indicative` is a free indicative feed where trades are delayed and quotes are aggregated.
```

Comment:
This wording is inaccurate. Instead of this, I'd just mention that `opra` requires a subscription and `indicative` is free.